keg_bouncer/model/migration.py

```diff
@@ -21,3 +24,68 @@ def link_user_to_user_groups(op, user_primary_key_column):
 
 def drop_link_from_user_to_user_groups(op):
     op.drop_table(user_user_group_link_table_name)
+
+
+def alembic_create_permission(op, permissions=[{}], groups=set(), bundles=set()):
+    """Bulk create permissions and add them to groups and bundles
+
+    :param permisisons: a list of dictionaries with 'token' and 'description' keys
+    :param groups: a set of group labels, missing groups will be created
+    :param bundles: a set of bundle labels, missing bundles will be created
+    """
+
+    new_permission_tokens = set([perm['token'] for perm in permissions])
+    groups = set(groups)
+    bundles = set(bundles)
+    con = op.get_bind()
+    meta = sa.MetaData()
+    execute = con.execute
+
+    # inspect the tables we need and create a table object for them
+    permissions_table = sa.Table('keg_bouncer_permissions', meta, autoload=True,
+                                 autoload_with=con.engine)
+
+    groups_table = sa.Table('keg_bouncer_user_groups', meta, autoload=True,
+                            autoload_with=con.engine)
+
+    groups_map = sa.Table('keg_bouncer_user_group_permission_map', meta, autoload=True,
+                          autoload_with=con.engine)
+
+    bundles_table = sa.Table('keg_bouncer_permission_bundles', meta, autoload=True,
+                             autoload_with=con.engine)
+
+    bundles_map = sa.Table('keg_bouncer_bundle_permission_map', meta, autoload=True,
+                           autoload_with=con.engine)
```

Comment:
Using ORM objects in migrations isn't the best idea, nor is it guaranteed consistent. For example, lets say you have a column name `label` that is created in migration b but removed in c. The orm field is also removed.  When the database is at migration a and tries to upgrade to c via b, migration b will fail because it wont find the column in the ORM object. 

However, if you inspect the table in the migration, you will get the picture of the table at that point in history and not what it is currently in the code.  

This is even true for helper functions like this, for example, migration _a_ creates a permission with this function using `label`, migration _b_ renames `label` to `other`, _c_ creates a permission with a new function. If we used orm objects, and the database was at a migration prior to _a_, the migration would fail because the orm field no longer exists. With tables, that isn't an issue.

Make sense? Did I miss some important detail?